Jenkinsfile.ad-hoc

```diff
@@ -131,7 +118,15 @@ def build(Map m){
     return {
         if (buildParam) {
             node(nodeLabel) {
-                getImageName().inside() {
+                getImageName().inside('-v "/home/jenkins/jenkins/workspace/Mobile/cliqz-android/ad-hoc\ builds/:/orig"') {
```

Comment:
Isn't it better to get the `pwd` instead of hard coding the directory ?

Reply:
yes it is